docker-compose.yml

```
@@ -0,0 +1,94 @@
+  fastapi:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: analytical-dashboard
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    networks:
+      - backend
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)